feature/bgp/policybase/otg_tests/import_export_multi/import_export_multi.go

```diff
@@ -0,0 +1,522 @@
+		} else if lossPct < 99 && !testResults[index] {
+			t.Errorf("FAIL- got %v%% packet loss for %s flow and prefixes: [%s, %s]; want >99%% traffic loss", lossPct, "flow"+"ipv4"+strconv.Itoa(index), prefixPairV4[0], prefixPairV4[1])
+		} else {
+			t.Logf("Traffic validation successful for Prefixes: [%s, %s]. Result: [%t] PacketsTx: %d PacketsRx: %d", prefixPairV4[0], prefixPairV4[1], testResults[index], txPackets, rxPackets)
+		}
+
+	}
+}
+
+func verifyTrafficV6(t *testing.T, bs *cfgplugins.BGPSession, testResults [6]bool) {
+
+	sleepTime := time.Duration(totalPackets/trafficPps) + 2
+	bs.ATE.OTG().StartTraffic(t)
+	time.Sleep(time.Second * sleepTime)
+	bs.ATE.OTG().StopTraffic(t)
+
+	otgutils.LogFlowMetrics(t, bs.ATE.OTG(), bs.ATETop)
+	otgutils.LogPortMetrics(t, bs.ATE.OTG(), bs.ATETop)
+
+	for index, prefixPairV6 := range prefixesV6 {
+		t.Logf("Running traffic test for IPv6 prefixes: [%s, %s]. Expected Result: [%t]", prefixPairV6[0], prefixPairV6[1], testResults[index])
+
+		t.Log("Checking flow telemetry...")
+		recvMetric := gnmi.Get(t, bs.ATE.OTG(), gnmi.OTG().Flow("flow"+"ipv6"+strconv.Itoa(index)).State())
+		txPackets := recvMetric.GetCounters().GetOutPkts()
+		rxPackets := recvMetric.GetCounters().GetInPkts()
+		lostPackets := txPackets - rxPackets
+		lossPct := lostPackets * 100 / txPackets
+
+		if lossPct > 1 && testResults[index] {
+			t.Errorf("FAIL- got %v%% packet loss for %s flow and prefixes: [%s, %s]; want < 1%% traffic loss", lossPct, "flow"+"ipv6"+strconv.Itoa(index), prefixPairV6[0], prefixPairV6[1])
+		} else if lossPct < 99 && !testResults[index] {
+			t.Errorf("FAIL- got %v%% packet for %s flow and prefixes: [%s, %s]; want >99%% traffic loss", lossPct, "flow"+"ipv6"+strconv.Itoa(index), prefixPairV6[0], prefixPairV6[1])
+		} else {
+			t.Logf("Traffic validation successful for Prefixes: [%s, %s]. Result: [%t]  PacketsTx: %d PacketsRx: %d", prefixPairV6[0], prefixPairV6[1], testResults[index], txPackets, rxPackets)
+		}
+
+	}
+}
+
+func TestImportExportMultifacetMatchActionsBGPPolicy(t *testing.T) {
+	bs := cfgplugins.NewBGPSession(t, cfgplugins.PortCount2, nil)
+	bs.WithEBGP(t, []oc.E_BgpTypes_AFI_SAFI_TYPE{oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST, oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST}, []string{
+		"port1", "port2"}, true, false)
+
+	configureOTG(t, bs, prefixesV4, prefixesV6, communityMembers)
+	bs.PushAndStart(t)
+
+	t.Log("Verify DUT BGP sessions up")
+	cfgplugins.VerifyDUTBGPEstablished(t, bs.DUT)
+	t.Log("Verify OTG BGP sessions up")
+	cfgplugins.VerifyOTGBGPEstablished(t, bs.ATE)
+
+	ipv4 := bs.ATETop.Devices().Items()[1].Ethernets().Items()[0].Ipv4Addresses().Items()[0].Address()
+	ipv6 := bs.ATETop.Devices().Items()[1].Ethernets().Items()[0].Ipv6Addresses().Items()[0].Address()
+
+	t.Logf("Verify Import Export Accept all bgp policy")
+	matchSetOptions := oc.BgpPolicy_MatchSetOptionsType_ANY
+	configureImportExportAcceptAllBGPPolicy(t, bs.DUT, ipv4, ipv6, matchSetOptions)
+
+	bs.ATETop.Flows().Clear()
```

Comment:
This can be removed, you don't have any flows yet

Reply:
updated